bin/yaml/julia.yaml

```diff
@@ -21,3 +22,7 @@ compilers:
           url: https://julialangnightlies-s3.julialang.org/bin/linux/x86_64/julia-latest-linux-x86_64.tar.gz
           strip_components: 1
           create_untar_dir: true
+    depot: {destination}/depot
+    cejl_revision: 532185e32ed03d340e16922c1da330c973047570
+    after_stage_script:
+      - JULIA_DEPOT_PATH={depot} {julia_exe} -e 'using Pkg; Pkg.add(; url="https://github.com/giordano/CompilerExplorer.jl", rev="{ce_revision}"); Pkg.precompile()'
```

Comment:
Is `bin/julia` (== `{julia_exe}`) a sensible way to call the julia executable after it has been installed?  I followed the `check_exe` example.